README.md

```diff
@@ -1,7 +1,94 @@
 # PCAxis.Serializers
-For the c# Serializers, see https://github.com/statisticssweden/PxWeb/issues/163
 
-# packagereference vs packages.config format, what to do?
-packagereference seems to be the future for non asp.net prosject
-We maintain asp.net stuff, so we have to know packages.config.
-packages.config gives the solution packages folder, which is clear
\ No newline at end of file
+This repository contains the serialization components required for serializing PCAxis data represented in PAXIOM from .px-files or CNNM-enabled databases. It enables the conversion from PAXIOM to various formats.
+
+## Supported Formats
+
+- Csv
+- DecimalJson
+- Excel
+- Html5Table
+- Json
+- Xml
+- Json-stat v1 & v2 (https://json-stat.org/format/)
+- Parquet (https://parquet.apache.org)
+- SDMX (https://sdmx.org/)
+- Xslx (https://learn.microsoft.com/en-us/openspecs/office_standards/ms-xlsx/f780b2d6-8252-4074-9fe3-5d7bc4830968)
+
+
+## Getting Started
+
+### Prerequisites
+
+- .NET Core 3.1 or later.
+
+### Installation
+
+```bash
+dotnet add package PCAxis.Serializers 
+```
+
+## Example Usage
+
+```csharp
+using System;
+using System.IO;
+using PCAxis.Paxiom;
+using PCAxis.Serializers;
+
+namespace MyPxConverter
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Please provide the PX filename as an argument.");
+                return;
+            }
+
+            string pxFileName = args[0];
+            string baseFileName = Path.GetFileNameWithoutExtension(pxFileName);
+            string outputFileName = $"{baseFileName}.parquet";
+
+            try
+            {
+                // Initialize and build the PX file model
+                var builder = new PXFileBuilder(pxFileName);
+                var selection = Selection.SelectAll(builder.Model.Meta);
+                builder.BuildForPresentation(selection);
+
+                // Serialize to Parquet
+                using (var stream = new FileStream(outputFileName, FileMode.Create))
+                {
+                    var parquetSer = new ParquetSerializer();
+                    parquetSer.Serialize(builder.Model, stream);
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex);
+            }
+        }
+    }
+}
+```
+
+## Contributing
+
+We welcome contributions to this project. Please see [CONTRIBUTING.md](CONTRIBUTING.md) for details.
```

Comment:
We do not have any CONTRIBUTING.md in this repository. Either we should add a copy of the one in the PxWeb repository or we should link to the CONTRIBUTIN.md file in the PxWeb repository

Reply:
Good catch 👍 We could probably just til to PxWeb for now

Reply:
Added the files since they don't change that often, and better not to cross-link.